examples/complete/main.tf

```diff
@@ -39,6 +39,7 @@ module "cis_dns_records" {
       ttl     = 900
     }
   ]
+  dns_records_file = null
```

Comment:
Can't we have default value as null in main...?
In that case, an user does not need to provide everytime this var and use this whenever required.

Reply:
Made the changes.